test/unit/feature_engineering/test_apply_features.py

```diff
@@ -105,7 +106,7 @@ def test_train_feature_applicator(self):
 
         self.assertDataframeEqual(
             input_data_with_features,
-            TestData.load("input_data_multi_horizon_features.csv"),
+            TestData.load("input_data_multi_horizon_features.csv").drop(columns=['is_bevrijdingsdag', 'is_bridgedaybevrijdingsdag']),
```

Comment:
It looks a little 'hacky' to drop these columns here.
Would it make more sense to change the 'apply_holidays_features' so the output columns are clearly defined?

Also, will this unit test bug pop-up again when we are in a year where 'bevrijdingsdag' is not an additional holiday?

Reply:
True, I will update the csv.

I checked and I think the holidays package changed some things as Bevrijdingsdag is not returned anymore for any year. I think this will not cause an issue anymore.